src/weekly/WeeklyExporter.js

```diff
@@ -101,7 +104,7 @@ export class WeeklyExporter {
     outputCsvRow[CUSTOMER_ADDRESS_LINE_2] = this.toUpperCase(row[CITY])
     outputCsvRow[CUSTOMER_ADDRESS_LINE_3] = this.formatState(row[STATE])
     outputCsvRow[CUSTOMER_POSTCODE] = this.toUpperCase(row[POSTAL_CODE])
-    outputCsvRow[DELIVERY_QUANTITY] = this.formatQuantity(row[QUANTITY])
+    outputCsvRow[DELIVERY_QUANTITY] = '1.0'
```

Comment:
👌🏻 (do we want to remove formatQuantity too?)

Reply:
I wasn't 100% sure we didn't need to get quantity from somewhere instead of just hard coding it to 1, but yeah I'll remove everything that is not being used and we can bring it back from the git history if we need to.